ts/api/api.ts

```diff
@@ -123,6 +124,11 @@ export interface OpenYoloApi {
    * Cancels the last pending OpenYOLO request.
    */
   cancelLastOperation(): Promise<void>;
+
+  /**
+   * Sets the desired log level for the OpenYOLO library.
+   */
+  setLogLevel(level: LogLevel): Promise<void>;
```

Comment:
If the `LogLevel` is sent through the message channel, unless we expose a method on `openyolo_spi`, the provider will not access the value. However, by using the query parameter technique, the provider's implementation will be able to read it from the URL.